examples/misc/font_indexing.py

```diff
@@ -1,28 +1,23 @@
 """
 =============
-Font Indexing
+Font indexing
 =============
 
-A little example that shows how the various indexing into the font
-tables relate to one another.  Mainly for mpl developers....
-
+This example shows how the font tables related to one another.
```

Comment:
This is not a correct English sentence, I think.

Reply:
Nothing wrong with the English, but it is referring to the past instead of the present:
```suggestion
This example shows how the font tables relate to one another.
```